docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -155,8 +155,13 @@ Usage
 .. code-block:: python
 
    from nni.algorithms.compression.v2.pytorch.pruning import SlimPruner
+   from nni.algorithms.compression.v2.pytorch.utils import trace
```

Comment:
Why is this in `algorithms`?
Suggest use another name rather than `trace` to avoid confusion.

Reply:
`trace` -> `trace_parameters`